ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml

```diff
@@ -59,6 +59,7 @@ resources:
 tests:
 - as: e2e-agent-compact
   cluster: build05
+  skip_if_only_changed: ^\.github/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|LICENSE)$
```

Comment:
is this parameter set on other repos as well? Just trying to understand why I runs more than e2e-assisted-metal and e2e-ai-operator-ztp in assisted-service repo.